react/react1/week1/src/ToDoList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import ToDo from './ToDo'
+
+
+export default function ToDoList({todos}) {
+  return (
+    todos.map(todo => {
+        return < ToDo key={todo.id} todo = {todo} /> 
```

Comment:
Good job, that you used key, why we use key see [here](https://reactjs.org/docs/lists-and-keys.html#keys)